RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/folding/PythonFoldParser.java

```diff
@@ -106,6 +112,15 @@ else if (leadingWhiteSpaceCount > currentLeadingWhiteSpaceCount) {
 		return folds;
 	}
 
+	private static 	boolean tokenHasLineContinuation(Token t) {
+		char ch = 'a';
+		t = t.getLastNonCommentNonWhitespaceToken();
+		if (t!=null && t.length()==1) {
+			ch = t.charAt(0);
+		}
+		return ch=='\\';
```

Comment:
This can be made a little simpler with [isSingleChar](https://github.com/bobbylight/RSyntaxTextArea/blob/9097e51fc5e289d761dca139a3a08459bf12614a/RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/Token.java#L445):

```java
t != null && t.isSingleChar('\\')
```

Reply:
OK.  Will commit with other changes.